generator/src/test/java/com/scottlogic/deg/generator/cucumber/ShorterThan.feature

```diff
@@ -220,42 +212,47 @@ Scenario: Running a 'shorterThan' request alongside a non-contradicting shorterT
        | "♀aaa" |
 
 @ignore
```

Comment:
Can we raise a bug on this or link to an existing?

Reply:
This is being addressed in ticket #501